src/styles/generator.scss

```diff
@@ -141,7 +141,7 @@ textarea:placeholder-shown + label.fancy-label {
     position: relative;
 
     .accordion-title {
-        padding-right: 30px;
+        padding: 3.1px 30px 0 0;
```

Comment:
Fractional pixels don't exist. Depending on the browser they might be treated differently: https://stackoverflow.com/questions/4308989/are-the-decimal-places-in-a-css-width-respected

Reply:
`3.1px` looked better in FF imo, but now I have changed it to `3px`